fluster/main.py

```diff
@@ -41,6 +42,11 @@ class Main:
 
     def __init__(self) -> None:
         self.decoders_dir = DECODERS_DIR
+        self.test_suites_dir = TEST_SUITES_DIR
+        # Only use the system directory for test suites if the local directory
+        # doesn't exist and the system directory does exist.
+        if not os.path.exists(TEST_SUITES_DIR) and os.path.exists(TEST_SUITES_DIR_SYS):
```

Comment:
If you want to apply this change only for Linux i believe the following will work without breaking compatibility with Windows and OSX.
```suggestion
        if sys.platform.startswith('linux') and not os.path.exists(TEST_SUITES_DIR) and os.path.exists(TEST_SUITES_DIR_SYS):
```